noxfile.py

```diff
@@ -8,8 +8,16 @@
 # Whenever type-hints are completed on a file it should be added here so that
 # this file will continue to be checked by mypy. Errors from other files are
 # ignored.
-STUB_FILES = {
+TYPED_FILES = {
```

Comment:
It's getting a little verbose to store both `.py` and `.pyi`, thoughts on adding a `.replace(".pyi", ".py")` to the end of the `filepath` logic to normalize everything to `.py`?

Reply:
Nice, very clean.